setup.py

```diff
@@ -35,20 +23,32 @@
     " permissions to connect to a Cloud SQL database without having"
     " to manually allowlist IPs or manage SSL certificates."
 )
-
-version = {}
-with open("google/cloud/sql/connector/version.py") as fp:
-    exec(fp.read(), version)
-version = version["__version__"]
-
 release_status = "Development Status :: 5 - Production/Stable"
-core_dependencies = [
+dependencies = [
     "aiohttp",
     "cryptography>=38.0.3",
     "Requests",
     "google-auth",
 ]
 
+package_root = os.path.abspath(os.path.dirname(__file__))
+
+readme_filename = os.path.join(package_root, "README.md")
+with io.open(readme_filename, encoding="utf-8") as readme_file:
+    readme = readme_file.read()
+
+version = {}
+with open(os.path.join(package_root, "google/cloud/sql/connector/version.py")) as fp:
+    exec(fp.read(), version)
```

Comment:
We will be removing `exec` from all libraries. This has already been added to `gapic-generator-python` and will be rolled out after the freeze. Please feel free to make the same change here or in a separate PR.

https://github.com/googleapis/gapic-generator-python/blob/2ad9be28500972777cec69097be6581eaefb89e4/gapic/templates/setup.py.j2#L22-L25

Reply:
Created issue for doing this in follow-up PR https://github.com/GoogleCloudPlatform/cloud-sql-python-connector/issues/909